.changeset/cyan-points-happen.md

```diff
@@ -0,0 +1,50 @@
+---
+'braid-design-system': minor
+---
+
+---
+updated:
+  - breakpoints
+  - useBreakpoint
+  - atoms
+  - responsiveStyle
+  - Accordion
+  - Box
+  - BulletList
+  - Columns
+  - Disclosure
+  - Heading
+  - Inline
+  - List
+  - MenuRenderer
+  - Stack
+  - Tabs
+  - Text
+  - Tiles
+---
+
+Add `wide` breakpoint of 1200px
+
+This adds support for `wide` to the following touchpoints:
+ - Responsive values, e.g.
+   ```ts
+   { mobile: 'small', wide: 'large' }
+   ```
+ - Responsive range props, e.g.
+   ```tsx
+   <Columns collapseBelow="wide" space={{ mobile: 'small', wide: 'large' }}>
+   ```
+ - The `responsiveStyle` function, e.g.
+   ```ts
+   export const className = style(responsiveStyle({ wide: '...' }));
+   ```
+ - The `breakpoints` object, which now exposes the number `1200` via `breakpoints.wide`, i.e.
+   ```ts
+   {
+     mobile: 0,
+     tablet: 740,
+     desktop: 940,
+     wide: 1200
+   }
+   ```
+ - The `useBreakpoint` Hook, which now returns `"wide"` on screens from 1200px and above.
```

Comment:
Doesn't this contradict [this statement](https://github.com/seek-oss/braid-design-system/pull/960/files#diff-71b58a179f50e429ac88569d49b52824e09495c71a8a563d8b708f6cd1f293ebR22)?

Reply:
Oops, nice catch 👍

Reply:
Writing this is actually what made me realise it was going to be a problem.